core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitUtils.java

```diff
@@ -158,21 +158,8 @@ public static TableMetadata removeTransactionalMetaColumns(TableMetadata tableMe
    * @return whether the specified column is a transactional meta column or not
    */
   public static boolean isTransactionalMetaColumn(String columnName, TableMetadata tableMetadata) {
```

Comment:
I refactored the logic of `isTransactionalMetaColumn()` and `isBeforeImageColumn()` and `isAfterImageColumn()`.